pkg/restore/restore_test.go

```diff
@@ -2113,6 +2112,130 @@ func TestRestorePersistentVolumes(t *testing.T) {
 	}
 }
 
+type fakeResticRestorerFactory struct {
+	restorer         *resticmocks.Restorer
+	podVolumeBackups []*velerov1api.PodVolumeBackup
+}
+
+func (f *fakeResticRestorerFactory) NewRestorer(ctx.Context, *velerov1api.Restore) (restic.Restorer, error) {
+	return f.restorer, nil
+}
+
+type fakeResticRestorer struct {
+	podVolumeBackups []*velerov1api.PodVolumeBackup
+}
+
+func (b *fakeResticRestorer) RestorePodVolumes(data restic.RestoreData) []error {
+	return nil
+}
+
+// TestRestoreWithRestic runs restores of only pods that contain associated PVBs.
```

Comment:
It would be nice to enhance the godoc comment to explain how the test/verification works (i.e. by using a mock for the restic restorer, and verifying that it's called for the expected pods).